contracts/SSVNetwork.sol

```diff
@@ -165,10 +165,11 @@ contract SSVNetwork is  UUPSUpgradeable, OwnableUpgradeable, ISSVNetwork {
         emit OperatorAdded(id, msg.sender, publicKey, fee);
     }
 
-    function removeOperator(uint64 id) external override {
-        Operator memory operator = _operators[id];
+    function removeOperator(uint64 operatorId) external override {
```

Comment:
@andrew-blox should `removeOperator` use `onlyOperatorOwnerOrContractOwner` modifier? thanks

Reply:
confirmed with product: `onlyOperatorOwner`